.clang-format

```diff
@@ -123,7 +128,7 @@ SpacesInCStyleCastParentheses: false
 SpacesInParentheses: false
 SpacesInSquareBrackets: false
 SpaceBeforeSquareBrackets: false
-# Needs new Clang: SpaceAroundPointerQualifiers: After
+SpaceAroundPointerQualifiers: After
```

Comment:
Since this has been disabled, it essentially isn't part of our style, and I'm not sure we should change it. I don't really understand how this option works either.

Reply:
Alright, I'll disable it. It basically adds a space after all qualifiers, for example `const`, disregarding the fact that pointer asterisks are left-aligned.